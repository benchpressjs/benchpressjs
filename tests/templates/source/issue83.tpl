
<!-- IF !thisIsNull --><!-- ENDIF !thisIsNull --> empty if causes compile error
