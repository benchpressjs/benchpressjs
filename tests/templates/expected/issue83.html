
 empty if causes compile error
